README.md

```diff
@@ -31,6 +31,12 @@ In this project, I was able to build my personal portfolio . The goal of this pr
 - Twitter: [@ShaddaiLinus](https://twitter.com/ShaddaiLinus)
 - Linkedin: [@ShaddaiLinus](https://www.linkedin.com/in/linusshaddai/)
 
```

Comment:
- [ ] Hello 👋 please remember that according to the project requirements 👨‍🏫 you should check all of the items listed in the article [Easy Checks – A First Review of Web Accessibility](https://www.w3.org/WAI/test-evaluate/preliminary/) and make a report according to your findings 

![image](https://user-images.githubusercontent.com/95243996/193809278-86919843-b124-4ddd-bc4e-14e7dbffc0c0.png)

Kyndly add your report to the description of your pull request.

Reply:
I'm sorry but I'm not sure I understand what you want me to do